.circleci/config.yml

```
@@ -157,6 +157,25 @@ jobs:
               --justfile ../../../single.just \
               simulate
 
+  just_simulate_006-MCP-L1:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.35.0
+
+    steps:
+      - checkout
+      - run:
+          name: just simulate 006-MCP-L1
+          command: |
+            just install
+            forge clean
+            forge build
+            cd tasks/eth/006-MCP-L1
+            export SIMULATE_WITHOUT_LEDGER=1
+            just \
+              --dotenv-path .env \
+              --justfile ../../../single.just \
+              simulate
```

Since this is a contract upgrade, the SC is involved, therefore the nested 2/2 safe is used and we need to specify who we are simulating with
```suggestion
              --justfile ../../../nested.just \
              simulate \
              council
```